### PR TITLE
Service Packs & AWS CIS Pack

### PR DESCRIPTION
**Pull request by wey-chiang**
Closes https://app.asana.com/0/1199940726973121/1200101326773347/f

### Background

This PR adds new packs based on the service provider as well as an AWS CIS pack

### Changes

* Added new packs which should have necessary globals included

### Testing

* @nhakmiller @lindsey-w Let's sync up to get this deployed to the test dev environment

### CODE REVIEW

File: packs/osquery.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+AnalysisType: pack
+PackID: PantherManaged.OSQuery
+Description: Group of all OSQuery detections
+PackDefinition:
+  IDs:
+    - Osquery.Linux.AWSCommandExecuted
+    - Osquery.Linux.LoginFromNonOffice
```
Comment (lindsey-w) [inline packs/osquery.yml:7]: Wondering how will we deal with "configuration required" detections? We want to manage them via packs, but should they be enabled with the non-configuration required detections?

File: packs/cisco_umbrella_dns.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+AnalysisType: pack
+PackID: PantherManaged.Cisco.Umbrella
+Description: Group of all Cisco Umbrella DNS detections
+PackDefinition:
+  IDs:
+    - CiscoUmbrella.DNS.Blocked
+    - CiscoUmbrella.DNS.FuzzyMatching
+    - CiscoUmbrella.DNS.Suspicious
+    # Globals used in these detections
```
Comment (lindsey-w) [inline packs/cisco_umbrella_dns.yml:9]: missing a global or copied comment?

File: packs/cisco_umbrella_dns.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+AnalysisType: pack
+PackID: PantherManaged.Cisco.Umbrella
+Description: Group of all Cisco Umbrella DNS detections
```
Comment (lindsey-w) [inline packs/cisco_umbrella_dns.yml:3]: Wondering if this should be the following (for any detections we add in the future that aren't `DNS` specifically)

```suggestion
Description: Group of all Cisco Umbrella detections
```

**Review by lindsey-w — COMMENTED** [review]
🥳

File: packs/cisco_umbrella_dns.yml (hunk 3 above)
Comment (wey-chiang) [reply packs/cisco_umbrella_dns.yml:3]: Makes sense to me

File: packs/cisco_umbrella_dns.yml (hunk 2 above)
Comment (wey-chiang) [reply packs/cisco_umbrella_dns.yml:9]: Copy pasta, left in if we ever add in detections that require globals

File: packs/osquery.yml (hunk 1 above)
Comment (wey-chiang) [reply packs/osquery.yml:7]: Huh, that's a good point -- had not considered the case for `configuration required`.

It'd be slick if we had a thing in the UI that specifically alerts users to detections requiring configuration

Comment (nhakmiller) [reply packs/osquery.yml:7]: It's a good question, typically we disabled by default detections that had the `configuration required` tag, but that doesn't make as much sense in the packs world. I think for now we should just include them,  and customers will need to know to search for that tag. In  the future we might have some CTAs in the UI  for these.

**Review by nhakmiller — APPROVED** [review]

**Pull request merged**